.dockerignore

```diff
@@ -9,6 +9,7 @@
 cache.sqlite
 README.md
 HACKING.md
+renovate.json
```

Comment:
I think you can also add:

```
Dockerfile
nginx.conf <- I don't know what it impacts when you do a `ADD nginx.conf .` so make sure I am not wrong :)
```

Reply:
I suspect we need `ngnix.conf` in the container.

Reply:
I can't add it to `./dockerignore` because the image won't build. But I can remove it in the production build